src/drive/mobile/containers/DriveMobileRouter.jsx

```diff
@@ -11,9 +11,10 @@ import { restoreCozyClientJs, initBar } from 'drive/mobile/lib/cozy-helper'
 class DriveMobileRouter extends Component {
   afterAuthentication = async ({ url, clientInfo, token, router }) => {
     const wasRevoked = this.props.isRevoked
+    this.context.client.options.uri = url
```

Comment:
`client.options.uri` is set at the start of the app, but the very first time the app starts, it's undefined. Since we use the variable in several places afterwards, it's a bit of a problem, so here we define it.